### ISSUE TITLE
Tiny equation fix

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#6476]
A pull request cleaned up a ton of documentation, IDD, and files. When @myoldmopar was trying to minimize the code and find a LaTeX compile problem, he messed up an equation.  To avoid burdening CI further right before IO freeze, he decided to make an issue out of it.

The problem is here: https://github.com/NREL/EnergyPlus/pull/6463#issuecomment-365701167

**Comment by RKStrand** [comment]
Issue that didn't get fixed and will get fixed here noticed by Bereket:

"Just check documentation in a local build. The equation for the curve Curve:ExponentialSkewNormal, in output reference on page 2270 is incorrect. It should have been Z2 = [(exp(C3x) * C4x) - C1] / C2, instead of C3, C4 was entered is used."

**Comment by RKStrand** [comment]
That's in the Input/Output Reference

**Comment by RKStrand** [comment]
@Myoldmopar Upon building what is in develop, the problem is slightly different than reported by Bereket, but there is definitely a problem here with Z2 and in general it isn't as easy to read as before.  I will clean it up so that it shows up as described in the V8.9 I/O Ref.

**Comment by RKStrand** [comment]
@Myoldmopar Ok, this is just plain weird.  I'm not 100% certain what is going on or why I/O Ref wasn't compiling properly.  When I made a new branch from develop and then compiled the I/O Ref, there was clearly a problem in the ExponentialSkewNormal equation for Z2.  However, when I made changes and then looked to see what GitHub was saying was different, it showed several other changes in other Exponential curve sections that I don't think I made.  Not sure why.  In any case, I have made changes and all of the equations in the different exponential curves in this section of the I/O Ref should be correct now but I'm not sure how it all arrived at that point and why the build results seemed to change.  I'll log a pull request, but feel free to ping me if something still seems off here.
